db-service/lib/SQLService.js

```diff
@@ -236,6 +236,18 @@ class SQLService extends DatabaseService {
       expand: false,
       limit: 0,
       orderBy: 0,
+      groupBy: 0,
+      ...(query.SELECT.groupBy
+        ? {
+            from: {
+              SELECT: {
+                columns: [{ ref: [query.SELECT.groupBy[0].ref[0]] }],
```

Comment:
This does not look dynamic enough. A GroupBy can have more then 1 column. And the ref can have more then 1 path entry.

Reply:
do we need to specify any column at all?
isn't a select 1 sufficient?

I'd guess that it's sufficient if we ignore any limit clause in the sub select as well.

Reply:
Wouldn't it make more sense to just wrap the original query (minus limiting factors) with a count.
```cds
SELECT count() FROM (query)
```

Reply:
Yes, I think that would be best.